includes/templates/admin-settings.php

```diff
@@ -136,3 +136,23 @@
 		</div>
 	</div>
 </div>
+
+<script type="text/javascript">
+	jQuery(function(){
+		const submitBtn = jQuery('#submit');
+		submitBtn.on('click', function(){
+			jQuery(this).addClass('accredible-button-spinner');
+		});
+		jQuery('#settings-form').on('submit', function(){
+			event.preventDefault();
+			const settings = jQuery(this).serialize();
+			jQuery.post('options.php', settings, function(res, status){
+				accredibleToast.success('Settings saved successfully', 3000);
```

Comment:
Are there any ways to update the state of the issuer info after the form submission? I updated the API key with an invalid value, but it kept saying "Integration is up and running".

<img width="1440" alt="Screen Shot 2022-04-08 at 10 40 49" src="https://user-images.githubusercontent.com/20009662/162347159-55110cee-55cf-46e0-80f5-decb871d844d.png">

Reply:
I think I'll have to update that to use Ajax as well. I have a few ideas on how to do that. Let me update it.

Reply:
This is now updated.